### PR TITLE
Folder added "Web based system"

### PR DESCRIPTION
**Pull request by anjaliasade**

### CODE REVIEW

File: anjali29/Web based system/appointment.html [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+      <div class="sidebar-overlay" data-reff="#side_menu"></div>
+      <script src="assets/js/jquery-3.2.1.min.js"></script>
+      <script src="assets/js/popper.min.js"></script>
+      <script src="assets/js/bootstrap.min.js"></script>
+      <script src="assets/js/select2.min.js"></script>
+      <script src="assets/js/moment.min.js"></script>
+      <script src="assets/js/bootstrap-datetimepicker.min.js"></script>
+      <script src="assets/js/theme.js"></script>
+   </body>
+</html>
```
Comment (poojakamble) [inline anjali29/Web based system/appointment.html:240]: remove new line

File: anjali29/Web based system/assets/js/theme.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
+
+
```
Comment (poojakamble) [inline anjali29/Web based system/assets/js/theme.js:2]: remove extra space

File: anjali29/Web based system/assets/js/theme.js [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+	
+	// Select2
+
+	if($('.select').length > 0 ){
+		$('.select').select2({
+			minimumResultsForSearch: -1,
+			width: '100%'
+		});
+	}
+});
```
Comment (poojakamble) [inline anjali29/Web based system/assets/js/theme.js:111]: remove new line

File: anjali29/Web based system/contact-us.html [hunk 4]
```diff
@@ -0,0 +1,260 @@
...
+      </footer>
+      <div class="sidebar-overlay" data-reff="#side_menu"></div>
+      <script src="assets/js/jquery-3.2.1.min.js"></script>
+      <script src="assets/js/popper.min.js"></script>
+      <script src="assets/js/bootstrap.min.js"></script>
+      <script type="text/javascript" src="https://maps.googleapis.com/maps/api/js"></script>
+      <script src="assets/js/theme.js"></script>
+      <script type="text/javascript">
+         google.maps.event.addDomListener(window, 'load', init);
+
```
Comment (poojakamble) [inline anjali29/Web based system/contact-us.html:233]: remove space

File: anjali29/Web based system/register.html [hunk 5]
```diff
@@ -0,0 +1,210 @@
...
+         </div>
+      </footer>
+      <div class="sidebar-overlay" data-reff="#side_menu"></div>
+      <script src="assets/js/jquery.min.js"></script>
+      <script src="assets/js/popper.min.js"></script>
+      <script src="assets/js/bootstrap.min.js"></script>
+      <script src="assets/js/owl.carousel.min.js"></script>
+      <script src="assets/js/theme.js"></script>
+   </body>
+</html>
```
Comment (poojakamble) [inline anjali29/Web based system/register.html:210]: remove new line

File: anjali29/Web based system/appointment.html (hunk 1 above)
Comment (JafarPathan) [inline anjali29/Web based system/appointment.html:240]: add new empty line at end for every html pages

**Pull request closed without merging**